src/sentry/event_manager.py

```diff
@@ -326,8 +326,6 @@ def normalize(self):
             except Exception:
                 # XXX: we should consider logging this.
                 pass
-            else:
-                data['tags'].extend(inst.iter_tags())
```

Comment:
Yes, this was accidentally left in which meant that tags in some circumstances were recorded twice.